usr/share/regolith-look/nevil/root

```diff
@@ -13,7 +13,7 @@ regolith.look: nevil
 ! --------------------------------------------
 
 gtk.theme_name: WhiteSur-light
-gtk.icon_theme_name: Arc
+gtk.icon_theme_name: Moka
```

Comment:
Looks good, but this also needs to be updated in the Debian package metadata so when the package is installed the Moka icon theme is installed if it's not already there.  This is the line you need to update: https://github.com/regolith-linux/regolith-look-extra/blob/main/debian/control#L59